tcms/telemetry/templates/telemetry/testcase-breakdown.html

```diff
@@ -0,0 +1,71 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+{% load comments %}
+{% load extra_filters %}
+{% load attachments_tags %}
+
+{% block title %}TestCase Breakdown{% endblock %}
```

Comment:
Use "Testing  Breakdown" as in the blog. Also needs to be marked for translation.